dll/win32/shell32/CDefView.cpp

```diff
@@ -1343,16 +1343,29 @@ LRESULT CDefView::OnCreate(UINT uMsg, WPARAM wParam, LPARAM lParam, BOOL &bHandl
     }
 
     SHChangeNotifyEntry ntreg[1];
-    ntreg[0].fRecursive = FALSE;
-    ntreg[0].pidl = m_pidlParent;
+    ZeroMemory(ntreg, sizeof(ntreg));
+
+    // Set up change notification
+    LPITEMIDLIST pidlTarget = NULL;
+    LONG events = 0;
```

Comment:
I would suggest you name it `fEvents` (like the parameter of the function), so that we understand it's a flag (or combination of flags), and _**NOT**_  a number of events....

Reply:
Fixed.